session/session_test.go

```diff
@@ -644,6 +644,10 @@ func testTxnLazyInitialize(s *testSessionSuite, c *C, isPessimistic bool) {
 func (s *testSessionSuite) TestGlobalVarAccessor(c *C) {
 	varName := "max_allowed_packet"
 	varValue := "67108864" // This is the default value for max_allowed_packet
```

Comment:
Please change it according to https://github.com/pingcap/tidb/pull/33651#discussion_r841070585